src/main/java/org/spongepowered/api/command/args/parsing/QuotedStringTokenizer.java

```diff
@@ -46,17 +46,15 @@
  * ARGS := ((UNQUOTED_ARG | QUOTED_ARG) WHITESPACE+)+</pre></blockquote>
  */
 class QuotedStringTokenizer implements InputTokenizer {
+    public static final QuotedStringTokenizer INSTANCE = new QuotedStringTokenizer(false);
+    public static final QuotedStringTokenizer LENIENT_INSTANCE = new QuotedStringTokenizer(true);
     private static final int CHAR_BACKSLASH = '\\';
     private static final int CHAR_SINGLE_QUOTE = '\'';
     private static final int CHAR_DOUBLE_QUOTE = '"';
-    private final boolean handleQuotedStrings;
     private final boolean forceLenient;
-    private final boolean trimTrailingSpace;
 
-    QuotedStringTokenizer(boolean handleQuotedStrings, boolean forceLenient, boolean trimTrailingSpace) {
```

Comment:
Is there any reason where `handleQuotedStrings` or `trimTrailingSpace` would be useful? I'm wondering if it's better to expose them, rather than remove them.

Reply:
`handleQuotedStrings` I can't speak on much. Given that the class is *called* `QuotedStringsTokenzier`, it seems like that kind of defeats the point a bit. I can't think of a use case for it and it wasn't being used, so there's no harm in removing it for now.

`trimTrailingSpaces` actually *needs* to be removed, as the tab completion system relies on the existince of an empty argument to separate a completed element from the start of a new one. Having this set to true would result in a variety of bugs there, hence removal.

Reply:
Ok.